src/feedback.cpp

```diff
@@ -222,8 +222,8 @@ std::string onProjectChange(Task& task, bool scope /* = true */) {
 std::string onProjectChange(Task& task1, Task& task2) {
   if (task1.get("project") == task2.get("project")) return onProjectChange(task1, false);
 
-  std::string messages1 = onProjectChange(task1);
-  std::string messages2 = onProjectChange(task2);
+  std::string messages1 = onProjectChange(task1, true);
+  std::string messages2 = onProjectChange(task2, true);
```

Comment:
The second argument defaults to `true` - any reason to pass it explicitly here?

Reply:
Thanks for your review! No reason here, so I corrected it.